### ISSUE TITLE
TorchLight "stuck on characters" - Error: TokenData lightAlpha "NaN" is not a number between 0 and 1

### ISSUE THREAD

**Issue by Gray-Geist** [PhilippeKr/TorchLight#40]
Hey all I keep getting this error "TokenData lightAlpha "NaN" is not a number between 0 and 1 which translates to a TorchLight module that works during the session but thereafter the light becomes "stuck as on" on my PCs/NPCs and only deleting the token and replacing it on the map seems to work. Any ideas? Sometimes deleting is a problem.

![image](https://user-images.githubusercontent.com/76698563/141221062-efea48da-bf89-4830-b85e-1b351f00076c.png)

@PhilippeKr can you help or direct Alan to help with this? 

Please note: I am not using DND 5e. All I am doing is setting the lights as on or off with the controls - I don't need it to auto detect things in inventory.

![image](https://user-images.githubusercontent.com/76698563/141221467-c0ba3688-713a-4b82-a1ed-686c79462edb.png)

**Comment by Gray-Geist** [comment]
Side note - even after disabling the mod, the torch light is still "stuck" on character tokens. Again, I can fix this by removing the token and replacing it w the same token but that is what I'm trying to avoid.

**Comment by Gray-Geist** [comment]
I have looked into this issue further and it may be related to #33 - where @eduardopato41 references a potential fix but it doesn't exactly apply to the new code. 

Requesting support to confirm if #33 truly is related, and if so how implement fix, if not how to fix the main issue here in #40

**Comment by Dadrak** [comment]
hello,
have you heard anything from the developper of the module? I always believed that after the last update the project was abandonned.

**Comment by Gray-Geist** [comment]
Hi there, unfortunately I have not - which is a shame - this mod is amazing. Are you going to take it up?

**Comment by Dadrak** [comment]
sadly I can't my coding knowledge is very limited, but I've read on discord that apparently he was going to update it. So lets hope, because it is very good

**Comment by Gray-Geist** [comment]
What Discord was that, the main FVTT one?

**Comment by Dadrak** [comment]
yes, on the FoundryVTT server, again I haven't hear it from him so not sure if it is true or not

**Comment by Gray-Geist** [comment]
Thanks for heads up either way, we can only hope!
